lustrefs-exporter/src/main.rs

```diff
@@ -540,7 +541,7 @@ mod tests {
     }
 
     fn compare_metrics(metrics1: &Scrape, metrics2: &Scrape) {
-        // Skip OTEL specific metric
+        // Skip OTEL specific metric and updated metrics.
         let set1: HashSet<_> = metrics1
             .samples
             .iter()
```

Comment:
Wondering if we should this test following #90 merge since this will fail for any updated metric

Reply:
Why will it fail now?

Reply:
Because we add new metrics but snapshot from previous impl are not updated with the new metric since old impl was removed